metadata-ingestion/src/datahub/ingestion/source/looker/looker_lib_wrapper.py

```diff
@@ -31,6 +32,11 @@
 logger = logging.getLogger(__name__)
 
 
+class LookerQueryResponseFormat(Enum):
+    JSON = "json"
+    SQL = "sql"
```

Comment:
minor: we may document here that the `sql` response format does not actually execute the query, just generates the SQL text query